spec/services/deals_service_spec.rb

```diff
@@ -27,8 +27,13 @@
 
   describe :create do
     it "returns instance of Deal class" do
-      @deal = build(:deal)
-      expect(client.deals.create(@deal)).to be_instance_of BaseCRM::Deal
+      deal = build(:deal)
```

Comment:
Don't you think that this change is not significant enough to add chaos to history of this file?